src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/X509Certificate.cs

```diff
@@ -500,34 +500,34 @@ public virtual string ToString(bool fVerbose)
 
         public virtual void Import(byte[] rawData)
         {
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(); // very low usage, and almost no one expects X509Certificate2 instances to be mutable
         }
 
         public virtual void Import(byte[] rawData, string password, X509KeyStorageFlags keyStorageFlags)
         {
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(); // very low usage, and almost no one expects X509Certificate2 instances to be mutable
         }
 
         [System.CLSCompliantAttribute(false)]
         public virtual void Import(byte[] rawData, SecureString password, X509KeyStorageFlags keyStorageFlags)
         {
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(); // very low usage, and almost no one expects X509Certificate2 instances to be mutable
         }
 
         public virtual void Import(string fileName)
         {
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(); // very low usage, and almost no one expects X509Certificate2 instances to be mutable
         }
 
         public virtual void Import(string fileName, string password, X509KeyStorageFlags keyStorageFlags)
         {
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(); // very low usage, and almost no one expects X509Certificate2 instances to be mutable
         }
 
         [System.CLSCompliantAttribute(false)]
         public virtual void Import(string fileName, SecureString password, X509KeyStorageFlags keyStorageFlags)
         {
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(); // very low usage, and almost no one expects X509Certificate2 instances to be mutable
```

Comment:
Should we have an explanatory message for these Imports? e.g. "Import is not supported on this platform due to X509Certificate being immutable".  And is there some alternative we could recommend in the exception message, to steer people towards the right solution instead?